components/raftstore/src/store/peer_storage.rs

```diff
@@ -205,18 +210,6 @@ impl EntryCache {
             }
         }
 
-        let (cache_len, mut entries_len) = (self.cache.len(), entries.len());
```

Comment:
Why? If cache are not expected to drop entries, better add comment explains the constraint.

Reply:
Has added a comment.